src/utils/github.ts

```diff
@@ -155,7 +155,34 @@ export class GithubApi {
   /**
    * Create a new pull request from the given branch and return pull request number
    */
-  async createPullRequest(branch: string, title: string): Promise<number> {
+  async createPullRequest(branch: string, title: string, botEmail: string, files: GithubFiles[]): Promise<number> {
```

Comment:
Would be nice to enhance the doc, like what is `botEmail`

Reply:
We can add extra documentation when we work on the changes to the data that goes into the Pull Request?

Reply:
I agree with Alice - this is something which was originally undocumented, so it doesn't really belong in a refactor PR.